tests/api/api/test_grafana_proxy.py

```diff
@@ -56,6 +56,9 @@ def test_grafana_proxy_model_endpoints_check_connection(
             )
         )
     )
+    mlrun.api.utils.clients.iguazio.Client().get_user_unix_id = unittest.mock.Mock(
+        return_value=0
+    )
```

Comment:
How this is related to grafana proxy model endpoints?

Reply:
there were some changes after I wrote this so I deleted this mock and added `user_unix_id=0` to the `verify_request_session` mock above.
The reason this is needed is that the request needs to be authenticated and if there is no unix id it will try to get it.